### DISCUSSION TITLE
Enable HTTPS verification message

### DISCUSSION THREAD

**Discussion by opentoe** [sabnzbd/sabnzbd#2184] [General]
I actually never touched that setting. I don't even use HTTPS since my WAN is behind a very tight firewall.

**Comment by thezoggy** [comment]
https://github.com/sabnzbd/sabnzbd/discussions/2182

**Comment by Safihre** [comment]
This setting is the one on Config > General page, it is not for connecting to SABnzbd but for when SABnzbd connects to a different websites, for example for fetch NZB's or RSS feeds.
